src/core/layout/grid/grid.js

```diff
@@ -157,6 +158,13 @@ export class GelGrid extends Phaser.GameObjects.Container {
         return this._cells.filter((cell, idx) => idx >= pageMin && idx < pageMax);
     }
 
+    getCellPage(theme, cellId = 0) {
+        const cellIdx = theme.choices.reduce((cellIdx, cell, idx) => {
+            return cell.id === cellId ? idx : cellIdx;
+        }, 0);
```

Comment:
Would findIndex work for this?
`const cellIdx = Math.Max(theme.choices.findIndex(cell => cell.id === cellId), 0);`

?

Reply:
yeah not sure why I didn't do that

Reply:
I guess it was the default value, I should use `Math.max` in these kinds of logic more often, very complimentary.